src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: $card-width;
+  padding: 16px;
+  margin-right: $distance-between-cards;
+  margin-bottom: 46px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 0;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $additional-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
```

Comment:
I think there are too many duplicate font values ​​in the project. Optionally, you can set the font size to a variable